### PR TITLE
Fix openapi.json to set allOf correctly

### PR DESCRIPTION
**Pull request by hsong-rh**
Fix `allOf` inheritance relationship in openapi json file

### CODE REVIEW

File: public/approval/v0.0.1/openapi.json [hunk 1]
```diff
@@ -1817,13 +1821,17 @@
                 "allOf": [
                     {
                         "$ref": "#/components/schemas/UserIn"
+                    },
+                    {
+                        "type": "object",
+                        "description": "Onput parameters for User object",
```
Comment (bzwei) [inline public/approval/v0.0.1/openapi.json]: In https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md I saw `"description"` is at top level parallel with `"offOf"`.

Comment (hsong-rh) [reply public/approval/v0.0.1/openapi.json]: Yes, the `description` will be automated skipped when creating client side docs.

Comment (bzwei) [reply public/approval/v0.0.1/openapi.json]: let's move to top for better readability.

Comment (hsong-rh) [reply public/approval/v0.0.1/openapi.json]: Changed.

**Pull request merged**